windows-observ-lib/panels.libsonnet

```diff
@@ -449,5 +449,327 @@ local utils = commonlib.utils;
+                                + fieldOverride.byName.withPropertiesFromOptions(
+                                  table.standardOptions.withUnit('ops')
+                                  + table.standardOptions.color.withMode('continuous-BlPu')
+                                ),
+                              ])
+                              + table.queryOptions.withTransformationsMixin(
+                                [
+                                  {
+                                    id: 'joinByLabels',
+                                    options: {
+                                      join: [
+                                        'instance',
+                                      ],
+                                      value: 'bind_method',
+                                    },
+                                  },
+                                  {
+                                    id: 'filterFieldsByName',
+                                    options: {
+                                      include: {
+                                        pattern: 'instance|digest|ds_client|ds_server|external|fast|ldap|negotiate|ntlm|simple',
+                                      },
+                                    },
+                                  },
+                                  {
+                                    id: 'organize',
+                                    options: {
+                                      renameByName:
+                                        {
+                                          instance: 'Instance',
+                                          digest: 'Digest',
+                                          ds_client: 'DS_client',
+                                          ds_server: 'DS_server',
+                                          external: 'External',
+                                          fast: 'Fast',
+                                          ldap: 'LDAP',
+                                          negotiate: 'Negotiate',
+                                          ntlm: 'NTLM',
+                                          simple: 'Simple',
+                                        },
+                                    },
+                                  },
+                                  {
+                                    id: 'reduce',
+                                    options: {
+                                      includeTimeField: false,
+                                      mode: 'reduceFields',
+                                      reducers: [
+                                        'last',
+                                      ],
+                                    },
+                                  },
+                                ]
+                              ),
+      intrasiteReplicationTraffic: commonlib.panels.generic.timeSeries.base.new(
```

Comment:
Missing unit (bps?).

Tip: Since it is a network traffic of specific type, I think you can try to use `commonlib.panels.network.timeSeries.traffic` as a closer prototype here.